docs_src/microservices/configuration/CommonConfiguration.md

```diff
@@ -0,0 +1,122 @@
+|Name      |----                       |Database or document store name (Specific to the service)            |
+|Timeout      |5000                           |DB connection timeout                                              |
+|Type |redisdb                        |DB type.  Redis is the only supported DB|
+
+
+!!! edgey "Edgex 2.0"
+    For EdgeX 2.0 `mongodb` has been remove as a supported DB. The credentials `username` and `password` have been removed and are now in the `Writable.InsecureSecrets.DB` section.
+
+
+
+## Registry
+
+|Property|Default Value|Description|
+|---|---|---|
+|||this configuration only takes effect when connecting to the registry for configuration info|
+|Host           |localhost                      |Registry host name|
+|Port           |8500                           |Registry port number|
+|Type           |consul                         |Registry implementation type|
+
+
+## Clients.[service-key]
+
+|Property|Default Value|Description|
+|---|---|---|
+|||Each service has it own collect of Clients that it uses|
+|Protocol | http  | The protocol to use when building a URI to local the service endpoint|
+|Host | localhost  | The host name or IP address where the service is hosted |
+|Port | 48081  | The port exposed by the target service|
+
+
+!!! edgey "Edgex 2.0"
+    For EdgeX 2.0 the map keys have changed to be the service's service-key, i.e. `Metadata` changed to `core-metadata`
+
+
+
+## SecretStore
+
+|Property|Default Value|Description|
+|---|---|---|
+|||these config values are used when security is enabled and Vault access is required for obtaining secrets, such as database credentials|
+|Host | localhost  | The host name or IP address associated with Vault|
+|Port | 8200  | The configured port on which Vault is listening|
+|Path | /`<service-key>` | The service-specific path where the secrets are kept. This path will differ according to the given service. |
+|Protocol | http  | The protocol to be used when communicating with Vault|
+|RootCaCertPath | blank | Default is to not use HTTPS |
+|ServerName | blank | Not needed for HTTP |
+|TokenFile | /tmp/edgex/secrets/`<service-key>`/secrets-token.json | Fully-qualified path to the location of the Vault root token. This path will differ according to the given service. |
+|AdditionalRetryAttempts | 10  | Number of attemtps to retry retrieving secrets before failing to start the service|
+|RetryWaitPeriod | 1s  | Amount of time to wait before attempting another connection to Vault|
+|Authentication AuthType | X-Vault-Token  | A header used to indicate how the given service will authenticate with Vault|
+
+
+
+!!! edgey "Edgex 2.0"
+    For EdgeX 2.0 the `Protocol` default has changed to `HTTP` which no loger requires `RootCaCertPath` and `ServerName` to be set. `Path` has been reduce to the sub-path for the service since the based path is fixed. `TokenFile` default value has changed and requires the `service-key` be used in the path.
```

Comment:
```suggestion
    For EdgeX 2.0 the `Protocol` default has changed to `HTTP` which no longer requires `RootCaCertPath` and `ServerName` to be set. `Path` has been reduce to the sub-path for the service since the based path is fixed. `TokenFile` default value has changed and requires the `service-key` be used in the path.
```